documentation/release_notes.rst

```diff
@@ -8,6 +8,56 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.9.0
+===============
+
+New Features
+------------
+- Added parallel range algorithms in ``namespace oneapi::dpl::ranges``: ``fill``, ``move``, ``replace``, ``replace_if``,
+  ``remove``, ``remove_if``, ``mismatch``, ``minmax_element``, ``min``, ``max``, ``find_first_of``, ``find_end``,
+  ``is_sorted_until``. These algorithms operate with C++20 random access ranges.
+- Improved performance of set operation algorithms when using device policies: ``set_union``, ``set_difference``,
+  ``set_intersection``, ``set_symmetric_difference``.
+- Improved performance of ``copy``, ``fill``, ``for_each``, ``replace``, ``reverse``, ``rotate``, ``transform`` and 30+
+  other algorithms with device policies on GPUs when using ``std::reverse_iterator``.
+- Added ADL-based customization point ``is_onedpl_indirectly_device_accessible`` which can be used to mark iterator
```

Comment:
Please add a comma between **``is_onedpl_indirectly_device_accessible``** and **which**.

Reply:
Done.